pages/index.md

```diff
@@ -1,14 +0,0 @@
----
-layout: home.html
-body_class: home
-title: Home
-plainlanguage:
-description: Apply for and manage the VA benefits and services you’ve earned as a Veteran, Servicemember, or family member—like health care, disability, education, and more.
-heading: Access and manage your VA benefits and health care
-fragments:
-  banner: home/banner
-  news: home/news
-  cards: home/cards
-  hubs: home/hubs
```

Comment:
These YML files can't be removed until the vets-website plugin is removed first, https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/apply-fragments.js

Reply:
Issue for that work, https://github.com/department-of-veterans-affairs/va.gov-team/issues/7640